From d0339fa10a29e78155a9d32a9c0ecf6cfda10a4d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Denis-Courmont?= <remi@remlab.net>
Date: Fri, 21 Oct 2016 15:56:51 +0300
Subject: [PATCH 1/5] Revert "Build default.vlt in a reproducible fashion
 (fixes #4562)"

This broke error propagation.

This reverts commit ceacef3bbb0c34d67aa84e6ca91b360cbe2419e5.

(cherry picked from commit 9f2f2165b7f7afa912fdf7ffbe8b3f0db51f687a)
---
 share/Makefile.am | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/share/Makefile.am b/share/Makefile.am
index 9894ebb..7f56458 100644
--- a/share/Makefile.am
+++ b/share/Makefile.am
@@ -77,10 +77,7 @@ skins2_default_vlt_FILES = \
 
 skins2/default.vlt: $(skins2_default_vlt_FILES)
 	mkdir -p skins2
-	(cd "$(srcdir)/skins2"; find default -print0 | \
-		LC_ALL=C sort -z | \
-		tar cvv --exclude .svn --no-recursion --null -T -) | \
-	gzip -n > skins2/default.vlt
+	tar cvvzf skins2/default.vlt -C $(srcdir)/skins2 default/
 
 #
 # LUA
-- 
2.10.2

